gecko/core/metrics.py

```
@@ -165,14 +216,27 @@ class MetricsRegistry:
+        初始化指标注册中心。
+        
+        参数：
+            max_label_combinations: 单个指标最多保留的标签组合数，防止内存泄漏。
+                                   当超过此限制时，最老的标签组合会被清理。
+        """
         self._counters: Dict[str, Counter] = {}
         self._gauges: Dict[str, Gauge] = {}
         self._histograms: Dict[str, Histogram] = {}
         self._lock = threading.Lock()
+        self.max_label_combinations = max_label_combinations
+        # 记录每个指标的最后一次使用时间，用于 TTL 清理
```

Ruff (0.14.7)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)